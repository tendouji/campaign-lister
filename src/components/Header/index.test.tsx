import React from 'react';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";
import { Header } from './';


const props = {
    title: 'Test Title',
};

describe('<Header />', () => {
    it('renders without crashing with props', () => {
        const wrapper = shallow(<Header {...props} />);
        expect(toJson(wrapper)).toMatchSnapshot()
    });
});


