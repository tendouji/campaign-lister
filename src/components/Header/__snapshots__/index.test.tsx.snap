// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crashing with props 1`] = `
<styled.header
  className="app-header"
>
  <div
    className="logo"
  />
  <h1>
    Test Title
  </h1>
</styled.header>
`;
